src/atoms/ScrollView/__tests__/ScrollView.web.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { renderWithTheme } from '../../../_helpers/testing';
+import ScrollView from '../index';
+
+describe('<ScrollView />', () => {
+  test('should match snapshot for ScrollView', () => {
+    const { container } = renderWithTheme(<ScrollView />);
+    expect(container).toMatchSnapshot();
+  });
```

Comment:
Let's follow the test suite we agreed upon

```suggestion
  it('renders Text', () => {
    const { container } = renderWithTheme(
    <ScrollView>
    <Text>Hello World</Text>
    </ ScrollView>
    );
    expect(container).toMatchSnapshot();
  });
```

Reply:
Added!